### PR TITLE
makes radio channels more visible on dark mode

### PR DESCRIPTION
**Pull request by nefarious6th**
## About the PR 
makes the radio station and medical channels have better visibility on dark mode

## Why's this needed? 
easier to see. honestly don't know why i didn't just make the med channel cyan in the first place.
radiocolors

### CODE REVIEW

**Comment by flrsh** [comment]
Could you include a screenshot? Thanks!

**Comment by nefarious6th** [comment]
> Could you include a screenshot? Thanks!

added to original PR body

**Comment by TobleroneSwordfish** [comment]
image
Very similar to this public intercom colour on cog 1

File: browserassets/css/browserOutput.css [hunk 1]
```diff
@@ -505,8 +505,8 @@ body.theme-dark .rcommand {color: #7ca6d6;}
 body.theme-dark .rsecurity {color: #e43535;}
 body.theme-dark .rdetective {color: #af4242;}
 body.theme-dark .rengineering {color: #dda448;}
-body.theme-dark .rmedical {color: #3A88AC;}
+body.theme-dark .rmedical {color: ##51DBCB;}
```
Comment (pgmzeta) [inline browserassets/css/browserOutput.css:508]: double hash->single hash

File: browserassets/css/browserOutput.css [hunk 2]
```diff
@@ -505,8 +505,8 @@ body.theme-dark .rcommand {color: #7ca6d6;}
...
 body.theme-dark .rresearch {color: #bf83e2;}
 body.theme-dark .rcivilian {color: #ce45b2;}
 body.theme-dark .rsyndicate {color: #cf5757;}
-body.theme-dark .rother {color: #bd46bd;}
+body.theme-dark .rother {color: ##E896E5;}
```
Comment (pgmzeta) [inline browserassets/css/browserOutput.css:512]: double hash -> single hash here

**Comment by pgmzeta** [comment]
Any chance you can include a pass at the syndicate radio channel while you're taking a look at these?

**Comment by nefarious6th** [comment]
> Any chance you can include a pass at the syndicate radio channel while you're taking a look at these?

ty ty for reviewing. i can give syndie radio a look and then update with that and a better medical color

**Review by ZeWaka — CHANGES_REQUESTED** [review]
^

**Comment by nefarious6th** [comment]
i will get back to this, sorry for the delay

**Comment by ZeWaka** [comment]
not necessary can be done later this is good now

**Pull request merged**